tools/omfwd.c

```diff
@@ -1484,6 +1494,8 @@ CODEmodInit_QueryRegCFSLineHdlr
 		NULL, &cs.iTCPRebindInterval, NULL));
 	CHKiRet(regCfSysLineHdlr((uchar *)"actionsendudprebindinterval", 0, eCmdHdlrInt,
 		NULL, &cs.iUDPRebindInterval, NULL));
+	CHKiRet(regCfSysLineHdlr((uchar *)"actionsendudpmulticastttl", 0, eCmdHdlrInt,
```

Comment:
Please remove the old-style config directives. We do not want to add any more of them.